src/js/themes/base.js

```diff
@@ -1462,6 +1462,23 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         // extend: undefined,
       },
     },
+    tag: {
+      // background: undefined,
+      border: true,
+      round: 'large',
+      name: { },
```

Comment:
```suggestion
      // name: undefined,
```
Should this be undefined if we don't have anything default in the theme?

Reply:
Whatever folks prefer. I put it as an empty object to make it clear it was like the value one but I suppose it would match some other examples.